xtable-hudi-support/xtable-hudi-support-extensions/pom.xml

```diff
@@ -112,6 +112,13 @@
             <artifactId>iceberg-parquet</artifactId>
         </dependency>
 
+        <!-- hudi -->
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-common</artifactId>
+            <scope>provided</scope>
```

Comment:
Does the produced jar function as expected after setting this as provided?

Reply:
This jar will be used by hudi users for generating iceberg and delta metadata, they would already have hudi-common in the class path.
https://github.com/apache/incubator-xtable/blob/main/xtable-hudi-support/xtable-hudi-support-extensions/src/main/java/org/apache/xtable/hudi/sync/XTableSyncTool.java

Reply:
https://github.com/apache/incubator-xtable/blob/main/xtable-hudi-support/xtable-hudi-support-extensions/README.md